pytorch_mnist/training/ddp/mnist/mnist_DDP.py

```diff
@@ -234,10 +235,10 @@ def run(modelpath, gpu):
   args = parser.parse_args()
   if args.gpu:
     logging.info("\n======= CUDA INFO =======")
-    logging.info("CUDA Availibility:", torch.cuda.is_available())
-    if (torch.cuda.is_available()):
-      logging.info("CUDA Device Name:", torch.cuda.get_device_name(0))
-      logging.info("CUDA Version:", torch.version.cuda)
+    logging.info("CUDA Availibility:", torch.cuda.is_available())  # pylint: disable = logging-too-many-args
```

Comment:
I think here will be have traceback confirm? also same below line.

Reply:
You are right, thanks for highlighting the typo.